Devin/feature/add burnwire component

dheadrick1618 opened the pull request.

rcunrau reviewed (changes requested):
It says 5000+ files changed :-(

rcunrau commented:
It might actually be easier to kill this PR and create a new one that only has what you want.

dheadrick1618 commented:
> It might actually be easier to kill this PR and create a new one that only has what you want.

How do I go about this? I have removed the build files and added a git ignore in the most recent commit, but those 5000+ changes are part of the commit history...

rcunrau commented:
I'm guessing you created 5-10 new files, and changed at most 1 or 2 existing files? The easiest thing is to do a new git clone in a different directory and copy over the new files. Make sure you make a new branch before committing, and specifically commit on the files you added/changed. The crazy history is on your current branch, and won't be on the new one.
